client/client.go

```diff
@@ -281,21 +281,97 @@ func newHttpClient() *http.Client {
 	return &http.Client{}
 }
 
+func loadServerTrust(ctx *openssl.Ctx, conf *Config) (*openssl.Ctx, error) {
+	// Trust CA's in the standard location (/etc/ssl/certs), and the
+	// configured server certificate when building the certificate chain
+	err := ctx.LoadVerifyLocations(conf.ServerCert, "/etc/ssl/certs/")
+	return ctx, err
+}
+
+func loadClientTrust(ctx *openssl.Ctx, conf *Config) (*openssl.Ctx, error) {
+
```

Comment:
do we need this extra line?

Reply:
Not really, but it is not uncommon for our code to have a little air here